### DISCUSSION TITLE
Disable inline commit info

### DISCUSSION THREAD

**Discussion by babakfp** [gitkraken/vscode-gitlens#1691] [Q&A] [answered]
Hi
How can I disable this:
![Screenshot 2021-10-17 011633](https://user-images.githubusercontent.com/44144724/137602870-29296e67-e1ed-49bb-9e84-e65a3467b311.png)
Thanks

**Comment by eamodio** [comment, marked as answer]
You can set `"gitlens.codeLens.enabled": false` in your settings.

You can also open the GitLens interactive settings editor from the command palette (`ctrl+shift+p` or `cmd+shift+p`) and search for _GitLens: Open Settings_

**Reply by babakfp** [reply]
Thanks
